addons/mail/models/discuss/discuss_call_history.py

```diff
@@ -0,0 +1,27 @@
+from odoo import api, models, fields
+from odoo.addons.mail.tools.discuss import Store
+
+
+class DiscussCallHistory(models.Model):
+    _name = "discuss.call.history"
+    _description = "Keep the call history"
+
+    channel_id = fields.Many2one("discuss.channel", index=True, ondelete="cascade")
+    duration = fields.Float(compute="_compute_duration", store=True)
```

Comment:
Could we specify the unit in the field name because this is confusing?
I think this is hours?
Why do we store hours as a float rather than number of seconds as int?

We can always convert when displaying it if necessary.

Reply:
`float_time` widget expects hours, so I don't really have a choice here.